tests/testcrate/tests/ui/sql_table.stderr

```diff
@@ -1,68 +1,100 @@
 warning: No primary key found
-  --> $DIR/sql_table.rs:34:8
+  --> $DIR/sql_table.rs:40:8
    |
-34 | struct Table<'a> {
+40 | struct Table<'a> {
    |        ^^^^^
 
 error: use of unsupported type name `& 'a str`
-  --> $DIR/sql_table.rs:36:13
+  --> $DIR/sql_table.rs:42:13
    |
-36 |     string: &'a str,
+42 |     string: &'a str,
    |             ^^^^^^^
 
 error: use of unsupported type name `Connection`
-  --> $DIR/sql_table.rs:38:17
+  --> $DIR/sql_table.rs:44:17
    |
-38 |     connection: Connection,
+44 |     connection: Connection,
    |                 ^^^^^^^^^^
 
 error: use of unsupported type name `Connection`
-  --> $DIR/sql_table.rs:40:25
+  --> $DIR/sql_table.rs:46:25
    |
-40 |     connection2: Option<Connection>,
+46 |     connection2: Option<Connection>,
    |                         ^^^^^^^^^^
 
 error: use of unsupported type name `Option<String>`
-  --> $DIR/sql_table.rs:42:28
+  --> $DIR/sql_table.rs:48:28
    |
-42 |     nested_options: Option<Option<String>>,
+48 |     nested_options: Option<Option<String>>,
    |                            ^^^^^^^^^^^^^^^
 
 error: use of unsupported type name `DateTime`
-  --> $DIR/sql_table.rs:44:15
+  --> $DIR/sql_table.rs:50:15
    |
-44 |     datetime: DateTime,
+50 |     datetime: DateTime,
    |               ^^^^^^^^
 
 error: use of unsupported type name `DateTime<i32>`
-  --> $DIR/sql_table.rs:46:19
+  --> $DIR/sql_table.rs:52:19
    |
-46 |     datetime_i32: DateTime<i32>,
+52 |     datetime_i32: DateTime<i32>,
    |                   ^^^^^^^^^^^^^
 
 error: use of unsupported type name `ForeignKey`
-  --> $DIR/sql_table.rs:48:20
+  --> $DIR/sql_table.rs:54:20
    |
-48 |     foreign_value: ForeignKey,
+54 |     foreign_value: ForeignKey,
    |                    ^^^^^^^^^^
 
 error: use of unsupported type name `Option`
-  --> $DIR/sql_table.rs:50:21
+  --> $DIR/sql_table.rs:56:21
    |
-50 |     optional_value: Option,
+56 |     optional_value: Option,
    |                     ^^^^^^
 
 error: use of unsupported type name `Vec`
-  --> $DIR/sql_table.rs:52:13
+  --> $DIR/sql_table.rs:58:13
    |
-52 |     vector: Vec,
+58 |     vector: Vec,
    |             ^^^
 
 error: use of unsupported type name `Vec<i32>`
-  --> $DIR/sql_table.rs:54:17
+  --> $DIR/sql_table.rs:60:17
    |
-54 |     vector_i32: Vec<i32>,
+60 |     vector_i32: Vec<i32>,
    |                 ^^^^^^^^
 
-error: aborting due to 10 previous errors
+error[E0106]: missing lifetime specifier
+  --> $DIR/sql_table.rs:40:8
+   |
+40 | struct Table<'a> {
+   |        ^^^^^ expected lifetime parameter
+
+error[E0107]: wrong number of type arguments: expected 0, found 1
+  --> $DIR/sql_table.rs:52:28
+   |
+52 |     datetime_i32: DateTime<i32>,
```

Comment:
We now have these errors in double.
`tql` used to emit them because they were not emitted by the compiler, but now it does not make sense to do so anymore.
I'll open an issue to remove them in the future.